site/docs/v1/tech/identity-providers/twitter.adoc

```diff
@@ -37,40 +37,35 @@ Once you have logged into Twitter and have obtained access to a Twitter develope
 
-=== Add the Callback URL
+=== Authentication Settings
 
-In this example I have registered `\https://login.piedpiper.com/oauth2/callback` as the callback URL.
+You will need to modify a few setting here to enable Twitter to connect with FusionAuth.
 
-The first part of that value will be equal to the public URL of FusionAuth, in this case `\https://login.piedpiper.com`. The second part, the actual redirect URI will always be `/oauth2/callback`.
-
-image::identity-providers/twitter-create-app.png[Create Twitter APp,width=1200,role=shadowed]
-
-
-=== Configure Permissions
+==== Add the Callback URL +++<span style"height: 50%; width: 50%;"><span class="fa-stack fa-2x" style="color: #F4821F;"><i class="fa fa-circle fa-stack-1x"></i><span class="fa-stack-1x calendar-text" style="color: black; font-family: sans-serif; font-size: medium">1</span></span>+++ & +++<span class="fa-stack fa-2x" style="color: #F4821F;"><i class="fa fa-circle fa-stack-1x"></i><span class="fa-stack-1x calendar-text" style="color: black; font-family: sans-serif; font-size: medium">2</span></span>+++
 
-Now you should have a Twitter app created, select the `Permissions` tab and click the `Edit` button in the top right of the panel.
+In this example I have registered `\https://login.piedpiper.com/oauth2/callback` as the callback URL. Please note that you will need to `enable` 3-legged OAuth and complete all required fields.
 
-You may select `Read-only` in the `Access permission` section and under `Additional permissions` ensure you have checked `Request email address from users`. An email address is required to complete login in FusionAuth.
+The first part of that value will be equal to the public URL of FusionAuth, in this case `\https://login.piedpiper.com`. The second part, the actual redirect URI will always be `/oauth2/callback`.
 
-If you will be using this Twitter app for other purposes, you may need more than `Read-only` permission, but for the purposes of configuring this Twitter App for FusionAuth only `Read-only` is required.
+==== Request Email +++<span class="fa-stack fa-2x" style="color: #F4821F;"><i class="fa fa-circle fa-stack-1x"></i><span class="fa-stack-1x calendar-text" style="color: black; font-family: sans-serif; font-size: medium">3</span></span>+++
 
-Click `Save` once you have completed these changes.
+Finally, you will want to ensure you have checked `Request email address from users.`  Please note, for some Twitter users, it is possible to create an account without an email address. If this is the case for your Twitter users, you will want to configure the IdP below to link on a username, or create a pending link. See link:/docs/v1/tech/identity-providers#linking-strategies[Linking Strategies]
 
-image::identity-providers/twitter-permissions.png[Twitter App Permissions,width=1200,role=shadowed bottom-cropped]
+image::identity-providers/twitter-create-app.png[Create Twitter APp,width=1200,role=shadowed]
```

Comment:
```suggestion
image::identity-providers/twitter-create-app.png[Create Twitter App,width=1200,role=shadowed]
```

Or remove, since you display it above.